src/main/java/adris/altoclef/Debug.java

```diff
@@ -1,17 +1,18 @@
 package adris.altoclef;
 
-import net.fabricmc.loader.launch.common.FabricMixinBootstrap;
-import net.fabricmc.loom.util.FabricApiExtension;
+
 import net.minecraft.client.MinecraftClient;
 import net.minecraft.text.Text;
 
+
 public class Debug {
+    // TODO: 2021-05-22 please use a logger
```

Comment:
Got it. Never used one before, I should look into it.

Reply:
yeah, I could introduce you to SLF4J or LOG4J if you want.